app/uk/gov/hmrc/helpfrontend/controllers/HelpController.scala

```diff
@@ -34,11 +34,11 @@ class HelpController @Inject() (
   termsAndConditionsPage: TermsAndConditionsPage,
   onlineServicesTermsPage: OnlineServicesTermsPage,
   cookiesPage: CookiesPage
-)(implicit val executionContext: ExecutionContext)
+)(using val executionContext: ExecutionContext)
```

Comment:
and these - if they're not referred to by name they can just be
```suggestion
)(using ExecutionContext)
```